packages/flutter_tools/test/general.shard/android/gradle_test.dart

```diff
@@ -546,9 +546,9 @@ flutter:
     });
   });
 
-  group('gradgradle_utils.le version', () {
+  group('gradle version', () {
```

Comment:
I genuinely wonder how I messed up 3 of these. sorry!

Reply:
It looks like it was probably a find and replace thing, and no worries considering I probably missed it in review :)